supabase-ui/src/index.js

```diff
@@ -8,3 +8,12 @@ export * from './components/Input/index'
 export * from './components/Radio/index'
 export * from './components/Select/index'
 export * from './components/Textarea/index'
+
+// OVERLAYS
+
+export * from './components/Modal/index'
+export * from './components/SideOver/index'
+
+// UTILITIES
+
+export * from './components/Transition/index'
```

Comment:
Great, nice to move this one in here too - would be cool to have a few little interactions we can use across our apps